NickvisionTubeConverter.Shared/Models/Configuration.cs

```diff
@@ -95,6 +90,8 @@ public Configuration()
         RunInBackground = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         SpeedLimit = 1024;
         UseAria = false;
+        AriaMaxConnectionsPerServer = 16;
```

Comment:
According to this: https://aria2.github.io/manual/en/html/aria2c.html

The default is 1.

Reply:
yt-dlp overwrites the default to 16 https://github.com/NickvisionApps/TubeConverter/issues/29#issuecomment-1514422019

Reply:
Ah ok :)